ui/components/ui/icon-with-fallback/icon-with-fallback.component.js

```diff
@@ -17,21 +18,28 @@ const IconWithFallback = ({
     setIconError(true);
   };
 
-  return !iconError && icon ? (
-    <img
-      onError={handleOnError}
-      src={icon}
-      style={style}
-      className={className}
-      alt={name || 'icon'}
-      {...props}
-    />
-  ) : (
-    <span
-      className={classnames('icon-with-fallback__fallback', fallbackClassName)}
-    >
-      {name && name.length ? name.charAt(0).toUpperCase() : ''}
-    </span>
+  return (
+    <div className={classnames(wrapperClassName)}>
+      {!iconError && icon ? (
+        <img
+          onError={handleOnError}
+          src={icon}
+          style={style}
+          className={className}
+          alt={name || 'icon'}
+          {...props}
+        />
+      ) : (
+        <span
+          className={classnames(
+            'icon-with-fallback__fallback',
+            fallbackClassName,
+          )}
+        >
+          {name && name.length ? name.charAt(0).toUpperCase() : ''}
```

Comment:
Could this be: `name?.charAt(0).toUpperCase() || ''`?

Reply:
this code was like the before, it is just the indenting that is changing

Reply:
done in 541c568749